javascript/javascript1/week4/voiceAssistant.js

```diff
@@ -0,0 +1,136 @@
+/*
+ * Javascript file for Hack Your Future, Javascript1, homework Week4
+ * program will return a response that corresponds to the command.
+*/
+
+// Voice assistant
+
+let userName = "";
+let toDo = [];
+
+function getReply(command) {
+
+    if (typeof command !== "string") {
+        return "Please retype your Question";
```

Comment:
Give an error message that says what is wrong, not just that something is wrong.
```suggestion
        return "Your question is not a string, please enter a question that is a string.";
```

Reply:
Thanks for the advice.